core/Makefile

```diff
@@ -34,14 +34,15 @@ include ../buildenv.mk
 App_Name := ehsm-core
 Enclave_Name := libenclave-$(App_Name).so
 Signed_Enclave_Name := libenclave-$(App_Name).signed.so
-
+Napi_Name := libehsmnapi.so
 Enclave_Config_File := Enclave/enclave.config.xml
 Enclave_Signing_Key := Enclave/enclave_private_test.pem
 
 OUT = $(TOPDIR)/$(OUTDIR)/$(App_Name)
 
 ######## App Settings ########
 App_Cpp_Files := $(wildcard App/*.cpp)
+App_Cpp_Files += $(wildcard App/lib_jsoncpp/*.cpp)
```

Comment:
change it as 
Napi_Cpp_Files += $(wildcard App/lib_jsoncpp/*.cpp)

Reply:
ignore the above comments,  otherwise the unit-test may be failed..

Reply:
The jsoncpp will be remove.